x-pack/test_serverless/functional/test_suites/observability/navigation.ts

```diff
@@ -35,7 +35,6 @@ export default function ({ getPageObject, getService }: FtrProviderContext) {
 
       // check side nav links
       await svlCommonNavigation.sidenav.expectSectionOpen('observability_project_nav');
-      await svlCommonNavigation.sidenav.expectLinkActive({ deepLinkId: 'observabilityOnboarding' });
```

Comment:
Removed this because nav group links don't get an active state set

Reply:
Not sure I understand. Your changes broke this functional test, is that what happened and you had to remove it?

Reply:
The test was asserting that the link had an active state, given that it's now represented as nav groups do not have an active state is why I removed it.

Reply:
I see. Thanks for the explanation 👍